e2e_test/iceberg/test_case/iceberg_engine.slt

```diff
@@ -15,6 +15,11 @@ select * from t;
 ----
 1 xxx
 
+query ??
+select * from t for system_time as of '2222-12-10 11:48:06';
```

Comment:
Does `'2222-12-10 11:48:06'` correspond to `7982020086000` or `7981991286000`?

Reply:
I look at the implementation, currently all timestamp string without explicit timezone will be considered to be UTC and don't respect the session timezone.

Reply:
Let's fix the current issue first and I will open another issue to track the timezone issue.

Reply:
https://github.com/risingwavelabs/risingwave/issues/19757